subsys/net/ip/net_shell.c

```diff
@@ -3532,6 +3532,173 @@ static int cmd_net_ipv6(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_net_ip6_add(const struct shell *shell, size_t argc, char *argv[])
+{
+#if defined(CONFIG_NET_NATIVE_IPV6)
```

Comment:
Again, if IPv6 is disabled I think the commands should print an error instead of going silent.

Reply:
Added info log.